tests/ro/creator/data/katoomba-rainfall/ro-crate-metadata.yaml

```
@@ -0,0 +1,39 @@
+"@blueprint":
+  extends:
+  glob: true
+
+"@context": "https://w3id.org/ro/crate/1.2/context"
+
+"@graph":
+
+"ro-crate-metadata.json":
+  "@type": "CreativeWork"
+  conformsTo: {"@id": "https://w3id.org/ro/crate/1.2"}
+  about: *root
```

YAMLlint (1.37.1)

[error] found undeclared alias "root"

(anchors)